teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person_class'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown')
+    super(name, age, parent_permission)
```

Comment:
- [ ] As the previous reviewer suggested, on line 5 you are setting `parent_permission`, but you did not define it on line 4. Kindly add it.

- [ ] Also ensure you arrange your arguments on line 5. Based on Person class, `age` comes before `name`

  ```
  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    ...
  ```